templates/Dataland/v2/index.html.twig

```diff
@@ -38,7 +38,7 @@
                             <div>
                                 <div>
                                     <div>
-                                        <strong>DOI:</strong> {{ dataset.doi.doi|default }}
+                                        <strong>DOI:</strong> {{ dataset.doi.doi|default|formatWebDoi }}
```

Comment:
Why not just format the URL here? Rather than using a whole function?

Reply:
yeah, you're right. We probably don't need to load more into twig.